### PR TITLE
MNT-21514  ACS-22:  Added new secondary cache map to avoid performance bottleneck in InMe…

### PR DESCRIPTION
**Pull request by Mykhail-Diachenko**
…moryTicketComponent in clustered environment

### CODE REVIEW

**Comment by CezarLeahu** [comment]
I'd be great we can get these changes reviewed & merged, as they are slowing down other work on _alfresco-data-model_ & _alfresco-repository_ (the chain/tree of dependencies).
The diffs can and should be viewed with the "_Hide whitespace changes_" GitHub feature active.

I can confirm that at least the Alfresco Admin Console is loading fine with these changes.

File: pom.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
 
         <maven.build.sourceVersion>11</maven.build.sourceVersion>
 
-        <dependency.alfresco-data-model.version>8.103</dependency.alfresco-data-model.version>
+        <dependency.alfresco-data-model.version>8.104</dependency.alfresco-data-model.version>
```
Comment (CezarLeahu) [inline pom.xml:39]: **8.105** is now the latest _alfresco-data-model_ release.

Comment (Mykhail-Diachenko) [reply pom.xml:39]: Thanks @CezarLeahu

**Comment by ancutam** [comment]
Please add the JIRA ticket id to the name of the pull request.

**Pull request merged**